lib/v2/qianzhan-analyst/column.js

```diff
@@ -0,0 +1,55 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    let rootUrl = 'https://www.qianzhan.com/analyst/';
+    const titles = {
+        all: '最新文章',
+        220: '研究员专栏',
+        627: '规划师专栏',
+        329: '观察家专栏',
+    };
+
+    if (ctx.params.type !== 'all') {
+        rootUrl = rootUrl + 'list/' + ctx.params.type + '.html';
+    }
+
+    const response = await got({
+        method: 'get',
+        url: rootUrl,
+    });
```

Comment:
got request can be simplify to
```suggestion
    const response = await got(rootUrl);
```